index-template.html

```diff
@@ -5,121 +5,58 @@
+      <li><a href="#">Config</a></li>
+    </ul>
+    <ul class="uk-switcher">
+      <li>
+        <viz-wiz config="#vizconfig"></viz-wiz>
+      </li>
+      <li>
+        <div id="visualizationinput" class="uk-container">
+          <pre><code id="configoutput"></code></pre>
+          <textarea id="vizconfig" style="visibility:hidden">
+{
+    "vizId": "123456",
+    "title": "What is my title",
+    "description": "This is a description",
+    "dataSources": [
+        {
+            "uid": "SomeRandomValue",
+            "type": "cob-arcgis",
+            "icon": "https://patterns.boston.gov/images/global/icons/mapping/waypoint-charles-blue.svg",
+            "clusterIcons": true,
+            "polygonStyle": "default",
+            "popover": "<p>I am a popover</p>",
+            "attributes": {
+                "service": "https://services.arcgis.com/sFnw0xNflSi8J0uh/ArcGIS/rest/services/food_trucks_schedule/FeatureServer",
+                "layer": 0
+            },
+            "legendLabel": "legendary"
         }
-      }
-      $(document).ready(function () {
-        var datatable, apiUrl = ''
-        document.addEventListener('changetable', function (event) {
-          var columnNames = []
-
-          event.detail.columns.forEach(function(item) {
-            columnNames.push(item.name)
-          })
-
-          apiUrl = event.detail.url + '?f=json&returnGeometry=false&outFields=' + columnNames.join(',')
-          datatable = setupTable(event.detail.columns, apiUrl)
-        })
-        document.addEventListener('myvizid.filter', function (event) {
-          $('#fevt').html('<code>' + JSON.stringify(event.detail) + '</code>')
-        })
-        document.addEventListener('changetable', function (event) {
-          console.log('ctevt')
-          $('#ctevt').html('<code>' + JSON.stringify(event.detail) + '</code>')
-        })
-        document.addEventListener('setfilter', function (event) {
-          $('#cfevt').html('<code>' + JSON.stringify(event.detail) + '</code>')
-        })
-        document.addEventListener('myvizid.filter', function (event) {
-          var queryParams = []
-          for (var field in event.detail) {
-            if (event.detail[field]) {
-              if (Array.isArray(event.detail[field])) {
-                queryParams.push(field + ' IN (\'' + event.detail[field].join('\',\'') + '\')')
-              } else {
-                queryParams.push(field + '=\'' + event.detail[field] + '\'')
-              }
-            }
-          }
-          if (queryParams.length === 0) {
-            queryParams = ['1=1']
-          }
-          var searchUrl = apiUrl + '&where=' + encodeURIComponent(queryParams.join(' AND '))
-          datatable.ajax.url(searchUrl).load()
-        })
-      })
+    ],
+    "maps": []
+}
+          </textarea>
+        </div>
+      </li>
+    </ul>
+    <script type="text/javascript">
+    var vizconfiginput = document.getElementById('vizconfig')
```

Comment:
Since this is the raw template/demo, I was uncomfortable using higher-level JS keywords since it is running directly in the browser, and not compiled for compatibility by Babel.

Reply:
Fair enough. In that case change the `let` below to `var`. (Though https://caniuse.com/#feat=const says everything we'd reasonably support for an internal tool can do `const`/`let`)